static/app/views/performance/landing/chart/histogramChart.tsx

```diff
@@ -102,12 +102,8 @@ export function HistogramChart(props: Props) {
             allSeries.push(series);
           }
 
-          const values = series.data.map(point => point.value);
-          const max = values.length ? Math.max(...values) : undefined;
-
           const yAxis = {
             type: 'value' as const,
-            max,
```

Comment:
@Zylphrex Would we need to do the same for the web vitals histograms? https://github.com/getsentry/sentry/blob/60d6126e9c3d8345ae13d76e35a55b5710add3c1/static/app/views/performance/transactionVitals/vitalCard.tsx#L263-L268

Reply:
No, the web vital histogram needs the max set in order to render that vertical baseline.